app.rb

```diff
@@ -19,9 +18,10 @@ class CondaAPI < Sinatra::Base
     package_version("pkgs/main", params[:name])
   end
 
-  get %r{/packages/(?<channel>.*)/(?<name>.*)/?} do
+  get "/packages/:channel/:name" do
+    channel = CGI.unescape(CGI.unescape(params[:channel]))  # %252F -> %2F -> /
```

Comment:
Dealing with a wsgi bug on flask, so making the `package` urlencoding be consistent everywhere